plutus-playground-server/plutus-playground-server.cabal

```diff
@@ -250,6 +243,3 @@ test-suite plutus-playground-server-test
         time-units -any,
         transformers -any,
         web-ghc -any
-
-    if !(impl(ghcjs) || os(ghcjs))
-        build-depends: plutus-tx-plugin -any
```

Comment:
I don't remember why but I have a feeling that there are might be consequences. Will see.